magento/adobe-stock-integration
# Added cancel button to Media Gallery

### Description

Added 'Cancel' button to Media Gallery to close the modal slide out.

### Fixed Issues

1. magento/adobe-stock-integration#908: Add "Cancel" button to the media gallery

### Manual testing scenarios 
1. Open Enhanced Media Gallery
2. Verify 'Cancel' button is present on the top right
3. Verify modal closes on button click
